Formula/vectorscan.rb

```diff
@@ -0,0 +1,39 @@
+class Vectorscan < Formula
+  desc "High-performance regular expression matching library"
+  homepage "https://github.com/VectorCamp/vectorscan"
+  url "https://github.com/VectorCamp/vectorscan/archive/refs/tags/vectorscan/5.4.7.tar.gz"
+  sha256 "cd70c2a7bf632b5374083a450019703605520c10c5614a4c12011c99ab8435dd"
+  license "BSD-3-Clause"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ragel" => :build
+  depends_on "pcre"
+
+
+  def install
```

Comment:
```suggestion
  depends_on arch: :arm64
  depends_on "pcre"

  def install
```
For now, while we have no `conflicts_with`.